deepquest/__init__.py

```diff
@@ -1,12 +1,12 @@
-def train(config=None, changes={}):
+def train(parameters):
```

Comment:
`deepquest.__init__.train()` should take a config path as an argument. This is the function that is called as `dq.train()`. Sorry if the structure is a little confusing, it's designed like this so that both the CLI and python module interfaces take similarly formed arguments.

Reply:
fixed with https://github.com/RSE-Sheffield/nmt-keras/pull/39/commits/b38be88cd4516f7d6b7c1787ece56827596feb6d.